.circleci/config.yml

```diff
@@ -1,13 +1,14 @@
 version: 2
 jobs:
-  build-iOS-Swift-4.2:
+  build-iOS-Swift-5.0:
     macos:
-      xcode: "10.0.0"
+      xcode: "10.2.0"
     environment:
           LC_ALL: en_US.UTF-8
           LANG: en_US.UTF-8
           HOMEBREW_LOGS: ~/homebrew-logs
           HOMEBREW_TEMP: ~/homebrew-temp
+          HOMEBREW_NO_AUTO_UPDATE: 1
```

Comment:
Need this as the CI image fo 10.2.0 is on MacOS 10.14.3. However, brew now required 10.14.4 to build packages such as swiftlint. This should be fixed on Circle CI side at some point in the near future by upgrading base image to 10.14.4